flagd/pkg/runtime/runtime.go

```diff
@@ -141,20 +145,34 @@ func (r *Runtime) updateAndEmit(payload sync.DataSync) bool {
 	r.mu.Lock()
 	defer r.mu.Unlock()
 
+	tp := otel.GetTracerProvider()
+	tracer := tp.Tracer("flagd-inprocess-sync")
+	ctx, span := tracer.Start(context.Background(), "flagd flagset update",
+		trace.WithSpanKind(trace.SpanKindProducer),
+		trace.WithAttributes(attribute.String("feature_flag.source", payload.Source)),
+		trace.WithAttributes(attribute.String("feature_flag.selector", payload.Selector)),
+		trace.WithAttributes(attribute.String("feature_flag.sync_type", payload.Type.String())),
```

Comment:
I think this attributes are very flagd specific, I'm not sure if we should perhaps use our own custom namespace for them?

Reply:
i like the idea, what is our preference here. Should it be `feature_flag.flagd.<attribute>` or simply `flagd.<attribute>`

Reply:
adapted based on my perception, please check if you agree with this format